cypress.config.js

```diff
@@ -1,9 +1,41 @@
 const { defineConfig } = require('cypress');
+const faker = require('faker');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com',
+    viewportHeight: 1320,
+    viewportWidth: 1080,
     setupNodeEvents(on, config) {
-      // implement node event listeners here
+      on('task', {
+        generateWorker() {
+          const ages = ['18', '19', '20', '21', '22', '23', '24', '25',
```

Comment:
what if here will be an array with 1000000 values?) do not need to create such test data, you could create a function that will generate random age between 18 and 89

Reply:
Done